webcompat/form.py

```diff
@@ -202,20 +215,23 @@ def build_formdata(form_object):
         'url': form_object.get('url'),
         'browser': form_object.get('browser'),
         'os': form_object.get('os'),
-        'problem_type': get_problem(form_object.get('problem_category')),
-        'description': form_object.get('description')
+        'problem_type': get_radio_button_label(
+            form_object.get('problem_category'), problem_choices),
+        'browser_test_type': get_radio_button_label(form_object.get(
+            'browser_test'), tested_elsewhere),
+        'description': form_object.get('description'),
+        'steps_reproduce': form_object.get('steps_reproduce')
     }
 
     # Preparing the body
     body = u'''{metadata}
 **URL**: {url}
+**Problem type**: {problem_type}
+**Description**: {description}
+**Steps to Reproduce** {steps_reproduce}
```

Comment:
I know it doesn't follow the format of the form, but I feel like STR shoul come last. It could contain a large image or content, which would mean you have to scroll down/up to see browser version, etc.

To be clear, I don't think we should change the flow of the reporting form, just the order once the report is made.

@karlcow @denschub @zoepage  thoughts?